packages/dapp/src/app/[locale]/app/(public)/qr-code-session/page.tsx

```diff
@@ -1,22 +1,12 @@
 import { Metadata } from 'next';
 
-import CreateConnectionCard from '@/components/CreateConnectionCard';
-import ScanConnectionCard from '@/components/ScanConnectionCard';
+import QRCodeScannerDisplay from '@/components/QRScannerDisplay';
 
 export const metadata: Metadata = {
-  title: 'Dashboard',
-  description: 'Dashboard for Masca Dapp.',
+  title: 'QR Code Scanner',
```

Comment:
Maybe change to QR Code Session ? Same thing in description and component name (QRCodeScannerDisplay).